__tests__/app.test.js

```diff
@@ -107,3 +107,48 @@ describe('/api/articles/:article_id', () => {
 		});
 	});
 });
+
+describe('/api/articles', () => {
+	describe('200: GET:', () => {
+		it('responds with status 200 and an array of object with the correct shape', () => {
+			const { commentData } = data;
```

Comment:
I think as you are asserting the sort order within this test it might be nice to give a nod to it in the test description